src/integration/kotlin/Kafka2Hbase.kt

```diff
@@ -68,4 +72,22 @@ class Kafka2Hbase : StringSpec({
         val counter = waitFor { hbase.getCount(topic) }
         counter shouldBe startingCounter
     }
+
+    "Malfomed messages are written to dlp topic" {
+        val topic = uniqueTopicName()
+
+        val body = "junk".toByteArray()
+        val timestamp = converter.getTimestampAsLong(getISO8601Timestamp())
+        producer.sendRecord(topic, "key3".toByteArray(), body, timestamp)
+
+        Thread.sleep(20000)
```

Comment:
Can't we loop here? The poll method below already has a pollTimeout, wouldn't that do instead of a sleep? If not, a loop with the poll inside and a max of 20 seconds?

Reply:
Actually realised that it doesn't need to be there..removed now